x-pack/plugins/security_solution/public/timelines/components/timeline/body/renderers/formatted_field.tsx

```diff
@@ -154,6 +161,10 @@ const FormattedFieldValueComponent: React.FC<{
         {content}
       </DefaultDraggable>
     );
+    if (fieldName === SIGNAL_STATUS_FIELD_NAME) {
```

Comment:
maybe we could add that as a next `else if` like https://github.com/elastic/kibana/pull/83963/files#diff-1220932fefda2a16462fd9a5aae85425fd158586bcb3002fecc766f23245b5e1R108?

Reply:
and extract that to a separate component, so we don't need to import `getSignalStatusBadge` logic here